editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2850,6 +2853,11 @@ void ThemeTypeEditor::_add_default_type_items() {
 	ur->commit_action();
 }
 
+void ThemeTypeEditor::_update_add_button(const String &p_text, LineEdit *p_for_edit) {
+	Button *button = Object::cast_to<Button>(p_for_edit->get_meta("button"));
+	button->set_disabled(p_text.strip_edges().is_empty());
```

Comment:
Since this logic of `strip_edges().is_empty()` is used relatively often it makes me wonder if it'd be worth it to add a convenience function that just checks if a string is non-empty after that operation, in this case if it has any codepoints `> 32`, will see about it later